PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -829,7 +829,7 @@ NodeType ProjectPanel::getNodeType(HTREEITEM hItem)
 		return nodeType_project;
 	}
 	// Folder
-	else if (tvItem.lParam == NULL)
+	else if (tvItem.lParam == 0)

```

Comment:
Semantic problem: it could be `nullptr` but not `0`.

Reply:
`LPARAM` is not a pointer but an integer. So it's not right to compare it against `NULL`. See my previous [comment](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/10048#discussion_r655494587).